nexus/src/bin/nexus.rs

```diff
@@ -49,5 +48,30 @@ async fn do_run() -> Result<(), CmdError> {
     let config = NexusConfig::from_file(config_path)
         .map_err(|e| CmdError::Failure(anyhow!(e)))?;
 
-    run_server(&config).await.map_err(|err| CmdError::Failure(anyhow!(err)))
+    let rt = {
+        let mut builder = tokio::runtime::Builder::new_multi_thread();
+        #[cfg(target_os = "illumos")]
```

Comment:
this _could_ also be conditional on `cfg(tokio_unstable)` so we don't fail to build if it's not set, but i actually _want_ builds to fail for now to make sure it's gettig set in the places I want it to be...